COORDINATE_SYSTEM2/COORDINATE_SYSTEM2/CoordinateSystem.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace COORDINATE_SYSTEM2
+{
+    internal class CoordinateSystem
```

Comment:
Shall we go through the requirements once again?
✔️   It should have method Add which takes Point object as an argument and another Add which takes double X and Y (method overloading)
✔️  It should be able to print all containing points (X and Y coordinate)
❌  It should be able to calculate distances between each point on the plane. 
❌  Find two points with the biggest/smallest distance.

Reply:
Why " ❌ It should be able to calculate distances between each point on the plane." - failed?
Let me make it more clear - I'm expecting to have a method `PrintAllDistances()` which just prints all distances in the following format "Distance between {p1} and {p2} is {distance}"

Reply:
Why "❌ Find two points with the biggest/smallest distance." - failed?
- it should have a method `PointsPair FindPointsWithMaxDistance()`
- it should have a method `PointsPair FindPointsWithMinDistance()`
As you can see, you need a new type PointsPair to hold two points, so you can return it as a result